include/message_filters/message_event.h

```diff
@@ -225,7 +213,7 @@ class MessageEvent
       return message_copy_;
     }
 
-    RCUTILS_ASSERT(create_);
+    assert(create_);
```

Comment:
`#include <cassert>` at the top of the file for this (and while we are at it, we may as well alphabetize that list of includes)